extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/ControllerImpl.java

```diff
@@ -1805,7 +1805,7 @@ private void handleQueryResults(
       Object resultObjectForStage = queryKernel.getResultObjectForStage(finalStageId);
       if (!(resultObjectForStage instanceof List)) {
         // This might occur if all workers are running on an older version. We are not able to write a manifest file in this case.
-        log.warn("Was unable to create manifest file due to ");
+        log.warn("Was unable to create manifest file. Expected list of file names from worker, received [%s] instead.", resultObjectForStage);
```

Comment:
Nit: slight rephrase:
```suggestion
        log.warn("Unable to create export manifest file. Received result[%s] from worker instead of a list of file names.", resultObjectForStage, finalStageId);
```